dbt/adapters/bigquery/connections.py

```diff
@@ -382,7 +382,7 @@ def get_bigquery_client(cls, profile_credentials):
         execution_project = profile_credentials.execution_project
         location = getattr(profile_credentials, "location", None)
 
-        info = client_info.ClientInfo(user_agent=f"dbt-{dbt_version}")
```

Comment:
We can no longer import the dbt-core version so will supply the dbt-bigquery one